docker/Dockerfile

```diff
@@ -21,7 +21,13 @@ RUN apt install -y -q \
     language-pack-en-base \
     yaru-theme-icon \
     shared-mime-info \
-    python3-pip
+    python3-pip \
+    python3-setuptools \
+    dcm2niix
```

Comment:
perhaps install this from their github release directly. see the instructions in this binaries section: https://github.com/ReproNim/neurodocker/blob/24d84b149b02e349de49398438299f9a2a712dec/neurodocker/templates/dcm2niix.yaml#L13

Reply:
is there specific reason we'd think the apt-gettable version isn't sufficient? i'd rather not increase the build time unless we have to.

Reply:
it's not always up to date. you can check how many revisions it lags by.

Reply:
also this shouldn't add any build time. did you see the instructions in the binaries section?

Reply:
ah gotcha. i guess my remaining question is whether there are specific concerns with a lagged version.

Reply:
i think for purpose of this PR less of a concern. this is a non-trivial conversion. i would worry if someone tries to use multi-echo data and more generally if this works well across scanners. for purpose of your target group this is less of a concern.

Reply:
that makes sense. i'll make an issue.